Packs/MySQLEnterprise/ModelingRules/MySQLEnterpriseEventCollector/MySQLEnterpriseEventCollector.yml

```
@@ -0,0 +1,4 @@
+name: MySQLEventCollector
+id: MySQLEventCollector
+fromversion: 3.3.0
```

Lets set this for debugging purpose and check if it causes the problem:
@yaakovi 

```suggestion
fromversion: 6.0.0
```